.github/workflows/pull-request.yml

```diff
@@ -1,8 +1,28 @@
 name: Pull Request
 
-on: [pull_request]
-permissions: read-all
+on:
+  pull_request:
+    types: [opened, edited, reopened]
+
+permissions:
+  pull-requests: read-all
+
 jobs:
   quality:
     name: Quality
     uses: ./.github/workflows/quality.yml
+
+  check-pr-title:
+    name: Check PR title
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
Can you try moving this checkout step to the Quality job, before line 7? I think we need this there to "pull in" our repo before running that Quality job.

Reply:
You mean before permissions?